locales/en/faq.json

```diff
@@ -94,9 +94,9 @@
       "question": "Where Can I Go to Learn More About Verge?",
       "answer": "We are on a ton of different social platforms and are very active on Twitter, Telegram, Slack, and Discord."
     },
-    "roadmap": {
-      "question": "Where Can I Find the Verge Roadmap?",
-      "answer": "We have a <a href='/roadmap' target='_blank'>roadmap</a> listed on our site that usually lists items we are working on for the next month or two. We don't like to put long-term items (greater than 3 months) on our roadmap, because things change too fast in crypto to anticipate that far ahead."
+    "milestones": {
```

Comment:
Then you need to adjust POEditor keys as well. I would just leave the key as roadmap

Reply:
Yep, they're in there twice